clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -697,6 +700,7 @@ public void handle(LeaveGroupResponse leaveResponse, RequestFuture<Void> future)
 
     // visible for testing
     synchronized RequestFuture<Void> sendHeartbeatRequest() {
+        log.debug("Sending Heartbeat request for group {} to coordinator {}", groupId, coordinator);
```

Comment:
Either works for me. The other requests are printed at DEBUG level. I don't see any harm in that since the heartbeats aren't sent _that_ often.